test/extension.test.ts

```diff
@@ -10,13 +10,6 @@ suite("Extension Tests", () => {
         assert.ok(vscode.extensions.getExtension(Constants.ExtensionId));
     });
 
-    test("should activate", function() {
```

Comment:
Extension could only be activated when extensionDependencies are installed, so remove it in CI.